dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/ZonedIntervalIterableTest.java

```diff
@@ -44,11 +44,17 @@ public class ZonedIntervalIterableTest {
   @SuppressWarnings("UnusedVariable")
   private static final Logger LOG = LoggerFactory.getLogger(ZonedIntervalIterableTest.class);
 
-  ConnectorArguments.ZonedParser zonedParserStart =
+  private static final TimeTruncator TRUNCATOR =
+      TimeTruncator.createBasedOnChronoUnit(ChronoUnit.SECONDS);
+
+  private static final ZonedDateTime START_TIME =
+      ZonedDateTime.parse("2007-12-03T10:15:30+01:00[Europe/Warsaw]");
```

Comment:
Appending `[Europe/Warsaw]` is optional. I would suggest to remove it here and in other places..